### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by ebinum83** [Ether1Project/Ether1#43]
I have been figuring out how to become a node operator and claim rewards. I used the docs site at **https://docs.ethoprotocol.com/nodes** The links that are in the docs on the staking pages **https://staking.ethofs.com**, times-out. and there is no login or register links on **https://ethoprotocol.com/get-started/staking**. Is this project dead?

**Comment by Legacytrx** [comment]
No, this project is definitely not dead, but we are revamping some bigger parts. This will address staking also. So be thrilled, because bigger things will come, but have some patience.
